src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4451,6 +4451,14 @@ components:
         description:
           type: string
           description: A human readable description of the contents of the Data Object.
+        md5:
+          type: string
+          description: >
+            A hex representation of the  md5 for the file.  If an md5 exists for the file being
+            ingested, it will be validated against this value and the file's ingest will fail if the
+            values do not match.  If no md5 hash is present on the file being ingested, this value
+            will be stored for the file.  If this value is not specified, then the md5 from the file
+            being ingested will be used stored
```

Comment:
I'm not following this last sentence, am I missing something?

Also, would this field be a good candidate to store as a ref?  No default value to contend with and I see we copy the definition in two places.

Reply:
derp...typo.  Yeah, let me move this to a ref